package/src/components/Message/MessageSimple/MessageContent.tsx

```diff
@@ -297,17 +303,21 @@ const MessageContentWithContext = <
             styles.containerInner,
             {
               backgroundColor,
-              borderBottomLeftRadius:
-                (groupStyle === 'left_bottom' || groupStyle === 'left_single') &&
-                (!hasThreadReplies || threadList)
-                  ? borderRadiusS
-                  : borderRadiusL,
-              borderBottomRightRadius:
-                (groupStyle === 'right_bottom' || groupStyle === 'right_single') &&
-                (!hasThreadReplies || threadList)
-                  ? borderRadiusS
-                  : borderRadiusL,
-              borderColor: isMyMessage && !error ? backgroundColor : grey_whisper,
+
+              borderBottomLeftRadius: shouldApplyBorderRadius('left_bottom', 'left_single')
```

Comment:
there is still unneeded duplication, let's just calculate and return either borderRadiusS or borderRadiusL